src/main/java/org/motechproject/mots/domain/Chiefdom.java

```diff
@@ -8,15 +8,21 @@
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 import lombok.Setter;
 
+@NoArgsConstructor
+@RequiredArgsConstructor
 @Entity
 @Table(name = "chiefdom")
 public class Chiefdom extends BaseEntity {
 
   @Column(name = "name", unique = true, nullable = false)
```

Comment:
I think we should remove unique here, it can have unique on name and district pair

Reply:
Do you foresee Chiefdom with duplicated names? There is none in XLSX source file

Reply:
there is none right now, but if we use this approach for community why not to replicate it for all the locations?

Reply:
not really needed, as chiefdoms are pretty solid, and not changing. But since I am at it, for the sake of simplicity let's go ahead